src/scss/components/_footer.scss

```diff
@@ -0,0 +1,191 @@
+                        font-weight: $font-weight-medium;
+                        line-height: $line-height--1-4;
+                        display: block;
+                    }
+                }
+            }
+        }
+
+        .locale-select-container {
+            padding: 0;
+            order: 1;
+            flex-shrink: 0;
+            margin-left: auto;
+
+            .form-group {
+                max-width: $max-width-600;
+            }
+        }
+
+        .form-label {
+            display: flex;
+            align-items: center;
+            margin: 0;
+            padding-bottom: $padding-bottom--8;
+            min-height: $min-height--28;
+        }
+
+        .sr-only {
+            position: absolute;
+            height: $height--1;
+            width: $width--1;
+            overflow: hidden;
+            clip: $form-label-clip;
+        }
+
+        .select-container {
+            position: relative;
+            min-width: $min-width--240;
+            max-width: $max-width-600;
+
+            .select-icon-container {
+                pointer-events: none;
+                display: flex;
+                align-items: center;
+                position: absolute;
+                top: 0;
+                height: $height-full;
+                padding: $padding--0--8;
+
+
+                &.select-icon-left {
+                    left: 0;
+                    justify-content: flex-end;
+                    width: $width--36;
+                }
+
+                &.select-icon-right {
+                    right: 0;
+                    width: $width--36;
+                }
+            }
+        }
+
+        .select {
+            background: $color-white;
+            border-radius: $border-radius--4;
+            border: $border-size-default solid $color-border-dark;
+            cursor: pointer;
+            display: block;
+            width: $width-full;
+            appearance: none;
+            -moz-appearance: none;
+            -webkit-appearance: none;
+            height: $height--40;
+            padding-left: $padding-left--36;
+            padding-right: $padding-right--40;
+        }
+
+        .logo-and-copyright {
+            display: flex;
+            justify-content: space-between;
+            align-items: center;
+            padding-top: $logo-and-copyright-padding-top;
+            padding-bottom: $logo-and-copyright-padding-bottom;
+
+            .logo-container {
+                display: flex;
+                align-items: center;
+                padding: 0;
+
+                .img-logo {
+                    max-width: $max-width-110;
+                }
+            }
+
+            .copyright-container {
+                padding: 0;
+                height: $height-full;
+                top: 50%;
+                color: $color-copyright;
+                font-weight: $font-weight-medium;
+                line-height: $line-height--1-4;
+                font-size: $font-size-sm;
```

Comment:
It's ok. But I think you should define 
.text-sm { font-size: $font-size-sm; }

We can use it every where when we need.

Reply:
Please define it in the `_typography.scss`